packages/react-signer/src/Modal.tsx

```diff
@@ -40,25 +45,31 @@ interface Props extends I18nProps, ApiProps, BaseProps {
 }
 
 interface State {
+  accountNonce?: string;
```

Comment:
Not 100% sure here why we have `accountNonce` and then `nonce` later on. It could be a hold-over from somewhere.

Reply:
This is for the hint in the nonce input. It shows the user the current nonce of the account and is the default value for `nonce` but you can change `nonce` to whatever you want